src/teaching-element/Audio.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="te-audio">
+    <aplayer
+      v-if="url"
+      v-bind="playerOptions"
+      :music="music" />
+  </div>
+</template>
+
+<script>
+import Aplayer from 'vue-aplayer';
+Aplayer.disableVersionBadge = true;
+
+const defaultAudioOptions = {
```

Comment:
Maybe this should be a function that returns an object.

Reply:
Any particular reason why?

Reply:
Nevermind.